src/pybind/mgr/cephadm/tests/test_services.py

```diff
@@ -269,7 +269,7 @@ def test_alertmanager_config(self, _run_cephadm, cephadm_module: CephadmOrchestr
                   webhook_configs:
                 - name: 'ceph-dashboard'
                   webhook_configs:
-                  - url: 'http://[::1]:8080/api/prometheus_receiver'
+                  - url: 'http://localhost:8080/api/prometheus_receiver'
```

Comment:
isn't this a valid test? The reconstructed dashboard url should probably check if the address is IPv4 or IPv6 re-add the square brackets to the hostname?